CONFIG.example

```diff
@@ -16,6 +16,9 @@ LM_LICENSE_FILE = 2100@diamcslicserv01.dc.diamond.ac.uk
 PANDA_ROOTFS = /home/mga83/targetOS/PandABlocks-rootfs
 # MAKE_ZPKG = $(PANDA_ROOTFS)/make-zpkg
 
+# Location of SDK used to find bootgen.
+SDK_ROOT = /dls_sw/apps/FPGA/Xilinx/SDK/2016.4
+
```

Comment:
why is this pointing to Vivado 2016.4?

Reply:
SDK_ROOT is only used for running bootgen, I suppose there is a better way.
I pointed to the same version that Panda rootfs is using so that both uses same bootgen

Reply:
This was removed